bitnami/metrics-server/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: metrics-server
-version: 4.2.2
+version: 4.2.3
```

Comment:
Please, bump the minor version (4.2.2 -> 4.3.0) as this change implements a new feature 😁

Reply:
Thanks, addressed!